src/solvers/fdsbp_tree/fdsbp_1d.jl

```diff
@@ -165,6 +165,14 @@ function calc_surface_integral!(du, u, mesh::TreeMesh{1},
     return nothing
 end
 
+# Periodic FDSBP operators need to use a single element without boundaries
+function calc_surface_integral!(du, u, mesh::TreeMesh1D,
+                                equations, surface_integral::SurfaceIntegralStrongForm,
```

Comment:
```suggestion
                                equations, surface_integral::Union{SurfaceIntegralStrongForm,SurfaceIntegralUpwind},
```
Instead of two times essentially identical implemenations? Same applies to the 2D/3D versions.

Reply:
Then it leads to method ambiguities since the implementations above specialize on `SurfaceIntegralStrongForm` or `SurfaceIntegralUpwind` (more specific for `surface_integral`) but we specialize on `dg` here. That's why we can't do that in a dimension-agnostic way - specialization on `TreeMesh1D` etc. requires the same specialization here

Reply:
OK 👍